### DISCUSSION TITLE
create a custom hook

### DISCUSSION THREAD

**Discussion by ahmed-abess** [directus/directus#13243] [Archive]
hello 
i want to create a costum hook that give me the ability of send  reset password email ,this mail contain diffirent url reset password related to role user for exemple admin get the url of reset password of directus back office  but the user get the url related to front office react interface

**Comment by rijkvanzanten** [comment]
Are you asking on how to create that hook currently, or are you requesting a new hook trigger that would fire on the "request a password reset" endpoint?

**Reply by ahmed-abess** [reply]
i ask for a solution any way if i should create a new endpoint or hook for this issue

**Reply by rijkvanzanten** [reply]
A new custom endpoint would be the way to go 👍🏻
